### DISCUSSION TITLE
Dashboard menu is not working in deployed website.

### DISCUSSION THREAD

**Discussion by HofSof** [OrchardCMS/OrchardCore#9747] [Q&A]
After deploying site, the menu in the dashboard is not showing.

Menu is displayed on development machine, but is not visible on hosted site.
Deploy was done by backup - restore database, and web deploy from VS to hosting. 
All the dashboard pages are working. Copy-pasting the urls from dev environment to hosted site opens the corresponding page.
And functions further normally  

 What could be the problem? Or how to get the menu working again.

**Comment by hishamco** [comment]
Could you please check if there's a console logs?

**Reply by HofSof** [reply]
No there is no log, if you mean log file in de App_data\logs..

**Reply by hishamco** [reply]
No no Console logs in browser

**Reply by Skrypt** [reply]
Did you enable the feature on the prod server? Could be that simple.

**Reply by hishamco** [reply]
Good catch, that's always may happen ;)

**Reply by HofSof** [reply]
In the first place appreciate you thinking with me.

Yes this was my first though. But all the features that are enabled on the development env. are set in production too.
And I think the dashboard menu is not a settable feature. If it is settable it should not be so.
It would render the dashboard and website unmanageable, 
This is how it looks like. 
![image](https://user-images.githubusercontent.com/79478277/122820035-75be2b00-d2db-11eb-8be0-521a6f58c63f.png)

**Reply by hishamco** [reply]
What about browser console logs (from Dev Tools F12)?

**Reply by Skrypt** [reply]
Looks like a permission not affected properly on the user role.
"Access the Admin Dashboard" permission maybe.

**Reply by HofSof** [reply]
No nothing in bowser console logs (F12)
And it is the admin user (administrator) created by the setup. So full access.
All pages are accessible 
Strange is it not.
